src/util/hass-mixins.html

```diff
@@ -104,9 +106,8 @@
       return hass && hass.resources;
     }
 
-    localize(namespace, message, ...args) {
-      // Return the input message if no translation is found
-      return super.localize(namespace + '.' + message, ...args) || message;
+    computeHaLocalize(localize) {
```

Comment:
I wonder if mixin is the right approach here,
This could be a function in utils - it doesn't need to be part of custom element prototype.
Then computeStateState could move back to utils too.

In any case, why introduce computeHaLocalize instead of overriding localize ?

Reply:
It needs to be at the component layer, because it's taking advantage of the Polymer app-localize-behavior, which only works because of the resources/language props which are part of the element state.

As far as why I couldn't just override localize, I'll copy my comment from above. Basically it boils down to the fact that Polymer's implementation works by rebinding the function when the resources become available, so in order to make use of that we need to do it the same way. If we also name our function `localize` here, we can't bind a computed property to the parent implementation.

> I hit a snag where translations weren't being updated after the resources were loaded. [I did some digging under the hood of the app-localize-behavior](https://github.com/PolymerElements/app-localize-behavior/blob/d3ff7ab958cc52a23b09c60faed55eced36dbc24/app-localize-behavior.html#L196), and found that the behavior `localize` function is actually a computed function (which is how the recalculation is triggered after resources are updated). In order to not break the polymer recalculation, I've updated our implementation to be a computed function as well. Unfortunately that meant I had to rename our child class method to not conflict with the behavior. Open to opinions on a better name, or if we consider dropping it altogether and just build the keys where we use them, since we're no longer using the extra behavior of falling back to a message.